qemu/tests/cfg/virtio_mem_hugetlb_migration.cfg

```diff
@@ -0,0 +1,36 @@
+- virtio_mem_hugetlb_migration:
+    only Linux
+    no Host_RHEL.m7 Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1 Host_RHEL.m9.u2 Host_RHEL.m9.u3
+    no RHEL.7 RHEL.8
+    no s390x
+    type = virtio_mem_hugetlb_migration
+    virt_test_type = qemu
+    smp = 8
+    vcpu_maxcpus = ${smp}
+    threshold = 0.025
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 8192
+    setup_hugepages = yes
+    mem_devs = 'mem0 vmem0'
+    backend_mem_vmem0 = memory-backend-file
+    mem-path_vmem0 = /mnt/kvm_hugepage
+    vm_memdev_model_vmem0 = "virtio-mem"
+    reserve_vmem0 = no
+    guest_numa_nodes = 'node0'
+    numa_memdev_node0 = mem-mem0
+    numa_cpus_node0 = "0-7"
+    size_mem_mem0 = 8G
+    use_mem_mem0 = no
+    size_mem_vmem0 = 8G
```

Comment:
As the `setup_hugepage` logic in the avocado-vt, it will use the size of `mem` to set up hugepage.  While in this case, the hugepage be used by `vmem0`.

It is tricky it can work now as the `vmem0` is equal to `mem` and also the `mem0` size,  and larger than the max `requested-size`,   but the logic is in correct.

Suggest:
size_mem_vmem0 = 8G
# workaround to setup correct hugepages based on 'mem'
mem = ${size_mem_vme0} 
size_mem_mem0 = ${mem}

Reply:
as the m`mem` value needs to be an int, I only updated the `size_mem_mem0` to be dependent on `size_mem_vmem0`